background/usw-api.js

```diff
@@ -0,0 +1,29 @@
+/* global URLS */ // toolbox.js
+
+'use strict';
+
+/* exported retrieveStyleInformation */
+async function retrieveStyleInformation(token) {
+  return (await (await fetch(`${URLS.usw}api/style`, {
+    method: 'GET',
+    headers: new Headers({
+      'Authorization': `Bearer ${token}`,
+    }),
+    credentials: 'omit',
```

Comment:
These credentials: `omit` are essentials parameters. As otherwise the cookies saved on these sites will also be send and currently the priority is given to cookies over headers. 

Making it so that it will return a error, as the cookies are in the `user` format which the api endpoint doesn't like.